scanner/image/scanner/konflux.Dockerfile

```
@@ -23,16 +23,16 @@ RUN make -C scanner NODEPS=1 CGO_ENABLED=${CGO_ENABLED} image/scanner/bin/scanne
 FROM ${BASE_REGISTRY}/${BASE_IMAGE}:${BASE_TAG}
 
 LABEL \
-    com.redhat.component="rhacs-scanner-container" \
+    com.redhat.component="rhacs-scanner-v4-container" \
     com.redhat.license_terms="https://www.redhat.com/agreements" \
-    description="This image supports image scanning for RHACS" \
+    description="This image supports v4 image scanning for RHACS" \
```

I'd suggest to move `v4` to reduce confusion.

```suggestion
    description="This image supports image scanning v4 for RHACS" \
```